src/modules/draft/draftQueries.ts

```diff
@@ -159,3 +163,40 @@ export const useDraftSearchTags = (params: UseSearchTags, options?: Partial<UseQ
     ...options,
   });
 };
+
+export interface UseGrepCodesSearch {
+  input: string;
+}
+
+interface GrepCodesSearchResult {
+  results: {
+    code: string;
+    title: string;
+  }[];
+  totalCount: number;
+  page: number;
+  pageSize: number;
+}
+
+export const useGrepCodesSearch = (
+  params: UseGrepCodesSearch,
+  options?: Partial<UseQueryOptions<GrepCodesSearchResult>>,
+) => {
+  return useQuery<GrepCodesSearchResult>({
+    queryKey: draftQueryKeys.grepCodesSearch(params),
+    queryFn: async () => {
+      const result = await fetchGrepCodes(params.input);
+      const convertedGrepCodes = await Promise.all(
+        result.results.map(async (c) => {
+          const grepCodeTitle = await fetchGrepCodeTitle(c);
```

Comment:
... Går det ikke an å gjøre dette kallet på en mer effektiv måte? Synes n+1 virker litt drastisk bare for å vise titler 🙃

Reply:
Helt sikkert, har ikke refakturert dette

Reply:
Håper du har fakturert det! Du kan godt refaktorere det altså, i og med at grep alltid har litt problemer :D